src/components/static/Countdown.jsx

```diff
@@ -65,37 +57,40 @@ function calculateBlocksForDuration(duration) {
  * @return {JSX.Element}
  */
 const Countdown = ({ targetTime }) => {
-  const [blocks, setBlocks] = useState(
-    calculateBlocksForDuration(Math.max(targetTime - Date.now(), 0))
-  );
+  const [blocks, setBlocks] = useState(calculateBlocksForDuration(0, 0));
   useEffect(() => {
-    const intervalId = setInterval(() => {
+    const update = () => {
       setBlocks(
         calculateBlocksForDuration(Math.max(targetTime - Date.now(), 0))
       );
-    }, 1000);
+    };
+    update();
+    const intervalId = setInterval(update, 1000);
```

Comment:
Run as soon as possible and every second. Without `update();` call, it takes at least a second before the countdown updates.

Reply:
this seems like badly styled code, what was wrong with the previous countdown? the one that is used throughout the other hackathons

Reply:
It waits an extra second before actually updating the countdown. setInterval doesn't run the callback immediately but runs it after the interval and every interval after that.